### DISCUSSION TITLE
Typos in PRO package

### DISCUSSION THREAD

**Discussion by dandarie** [Laravel-Backpack/community-forum#359] [General]
Hi.

I've found a bunch of typos in the language files and in the bulk_delete.blade.php file.

It seems that all over in these files _success_ was written as _sucess_.

```
↳ grep -r _sucess_ src/resources/lang/en/
src/resources/lang/en/crud.php:    'bulk_delete_sucess_title'   => 'Entries deleted',
src/resources/lang/en/crud.php:    'bulk_delete_sucess_message' => ' items have been deleted',
src/resources/lang/en/crud.php:    'bulk_clone_sucess_title'   => 'Entries cloned',
src/resources/lang/en/crud.php:    'bulk_clone_sucess_message' => ' items have been cloned.',

↳ grep -r _sucess_ . | wc -l
86
```

The messages are used by the PRO package.

If I were to submit a PR to the base repository, how can I make sure the changes are picked by the PRO team?

**Comment by pxpm** [comment]
Hey @dandarie thanks for raising your concerns.

We are aware of that issue, unfortunately to fix it without breaking change we would need to keep those erroneous keys and add the new ones with the fixed spelling in all language files. I don't think it's worth it now since v6 is due in a few months, we can evaluate doing it there and mention it in the upgrade guide. 

I will create an issue for this so that we have a look at it later. 

Cheers

**Reply by dandarie** [reply]
This reminds me of [HTTP referer](https://en.wikipedia.org/wiki/HTTP_referer) :)

**Comment by tabacitu** [comment]
Sorry guys, but the fix is not worth it. Let's keep the mistake.
